File: app/src/main/java/com/artmurka/artmurkaapp/Model/InterfacesModel/IAboutGoods.java
package com.artmurka.artmurkaapp.Model.InterfacesModel;

import com.artmurka.artmurkaapp.Model.Pojo.ItemList.AboutGoods.AboutGood;
import com.artmurka.artmurkaapp.Model.Pojo.ItemList.Good.Good;
import com.artmurka.artmurkaapp.Model.Pojo.ItemList.ItemList.SuccessExample;

import io.reactivex.Observable;
import retrofit2.Call;

public interface IAboutGoods {
    Call<AboutGood> getDataAboutGood(String id);
    Call<Good> getDataGood(String id);
    Observable<SuccessExample> getItemList(String page, String pageNumber);
}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/interfacesmodel/IAboutGoods.kt
package com.artmurka.artmurkaapp.data.model.interfacesmodel

import com.artmurka.artmurkaapp.data.model.pojo.itemlist.aboutgoods.AboutGood
import com.artmurka.artmurkaapp.data.model.pojo.itemlist.good.Good
import com.artmurka.artmurkaapp.data.model.pojo.itemlist.itemlist.SuccessExample

import io.reactivex.Observable
import retrofit2.Call

interface IAboutGoods {
    fun getDataAboutGood(id: String): Call<AboutGood>
    fun getDataGood(id: String): Call<Good>
    fun getItemList(page: String, pageNumber: String): Observable<SuccessExample>
}
